ddl/resourcegroup/group.go

```diff
@@ -30,10 +36,14 @@ func NewGroupFromOptions(groupName string, options *model.ResourceGroupSettings)
 	if len(groupName) > MaxGroupNameLength {
 		return nil, ErrTooLongResourceGroupName
 	}
+
 	group := &rmpb.ResourceGroup{
 		Name: groupName,
 	}
 	if options.RURate > 0 {
+		if options.RURate > MaxRUFillRate || options.RURate < MinRUFillRate {
```

Comment:
I think we don't need to limit the MaxRUFillRate and MinRUFillRate. cc @glorv

Reply:
+1